web/src/components/index.js

```diff
@@ -0,0 +1,28 @@
+import WilcoxonPlotTile from './vis/WilcoxonPlotTile.vue';
+import AnovaTableTile from './vis/AnovaTableTile.vue';
+import { wilcoxon_zero_methods, wilcoxon_alternatives, plot_types } from '../utils/constants';
```

Comment:
I prefer to have multiple `constants.js` files since with a growing application it will get pretty large, even tho certain elements (like `wilcoxon_zero_methods`) is only used one in the application

Reply:
Sure, that's fine.  Could you take a look the constants file and enumerate how you'd like to break that up?  How many constants files will we end up with, and what will their paths be?

Reply:
I prefer the localization principle. So, if imported in a single file -> but it directly in the file. Otherwise in a `constants.js` in the most common directory level where it will be imported. e.g. the `wilcoxon_zero_methods` would end up in `./vis/constants.js`

Reply:
In that case, it would probably be best to discuss that in another PR.  That refactor doesn't really affect any of the concerns of this PR.  Would that be alright?

Reply:
Sounds like we should file an issue around refactoring constants.

Reply:
-> https://github.com/girder/viime/issues/225